src/index.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-use-before-define */
+const api = {
+  key: '4cc8a926413ba21a3774022235df5269',
+  base: 'https://api.openweathermap.org/data/2.5/',
+};
+
+const searchbox = document.querySelector('.search-box');
+searchbox.addEventListener('keypress', setQuery);
+
+function setQuery(evt) {
+  if (evt.keyCode === 13) {
+    getResults(searchbox.value);
+  }
+}
```

Comment:
**This issue is reapeated through all your project, please implement this changes in all your functions**
- [ ] **ES6 Syntax:** 
  - [ ] Usage of arrow functions instead of the `function` keyword
  - [ ] Use Destructuring variables
**This function should look as following:**

```javascript
const setQuery = ( { keycode } ) => {
  if (keycode === 13) {
    const { value } = searchbox
    getResults(value)
  }
}
```